app/infra/eventsService/eventsService.ts

```diff
@@ -16,9 +16,9 @@ class EventsService {
 
   static getOsThemeColor = () => ipcRenderer.invoke(ipcConsts.GET_OS_THEME_COLOR);
 
-  static openBrowserView = () => ipcRenderer.send(ipcConsts.OPEN_BROWSER_VIEW);
+  static switchTheme = ({ isDarkMode }: { isDarkMode: boolean }) => ipcRenderer.send(ipcConsts.SET_OS_THEME_COLOR, { isDarkMode });
```

Comment:
```suggestion
  static switchTheme = ({ isDarkMode }: { isDarkMode: boolean }) => ipcRenderer.send(ipcConsts.SET_THEME_COLOR, { isDarkMode });
```
Proper name. We can not mutate the OS theme.